docset/windows/appx/add-appxpackage.md

```diff
@@ -313,7 +313,7 @@ Accept wildcard characters: False
 ```
 
 ### -RegisterByFamilyName
-{{Fill RegisterByFamilyName Description}}
+Specifies the parameter -MainPackage that define the family name or full name to register.

```

Comment:
```suggestion
Specifies the parameter -MainPackage that defines the family name or full name to register.
```

Reply:
@get-itips , I guess it is correct as it is one of the parameter.

-I tried to run cmd "Add-AppXPackage -RegisterByFamilyName -MainPackage {PackageFamilyname / Packagefull name} " > works successfully

-But without mainpackage, it will still ask for the -MainPackage name
<img width="630" alt="mainpackage" src="https://user-images.githubusercontent.com/48753349/63900678-e5db2d00-ca33-11e9-9572-7e790606baa5.PNG">
<img width="553" alt="with mainpackage" src="https://user-images.githubusercontent.com/48753349/63900689-ec69a480-ca33-11e9-961b-4f1dde78aaf3.PNG">

Reply:
Hi @get-itips
-RegisterByFamilyName, It specifies the parameter "-MainPackage" that define the family name , I guess they work together.

If  we run the cmd: "Add-AppXPackage -MainPackage" > this won't work (see screenshot above)
If  we run the cmd: "Add-AppXPackage -RegisterByFamilyName" it will ask you for the "-MainPackage"

We can view family name of the package by running the cmd "Get-Appxpackage -Allusers" :
See screenshot below:
<img width="626" alt="packagename" src="https://user-images.githubusercontent.com/48753349/63984282-1c30af00-cafd-11e9-96d6-983f0653c5ab.PNG">

Reply:
@e0i , see answer above with regard to "-MainPackage" , feel free to let me know of any changes needed.

Reply:
ok @sukihanabi thanks